src/progress-indicator/progress-indicator.component.ts

```diff
@@ -74,6 +75,7 @@ export class ProgressIndicator implements OnChanges {
 	@Input() steps: Array<Step>;
 	@Input() orientation: "horizontal" | "vertical" = "horizontal";
 	@Input() skeleton = false;
+	@Input() spaceEqually = false;
```

Comment:
I'm somewhat on board with that, though I do wonder what other possible values we could have for `space`? Also, perhaps `spacing = "equal"` (semantics I know 😛)

Reply:
Looking at how it fits together, it really seems like `space--equal` is a bit of a hack to fix the progress indicator behaviour ... I'm not sure that it entirely matters what we choose, since it seems likely to change

Reply:
maybe `default | equal` for now?

Reply:
Sure 👍 I'm good with whatever seems reasonable to everyone